shuup/admin/templates/shuup/admin/orders/_order_status_history.jinja

```diff
@@ -0,0 +1,71 @@
+<div class="d-sm-none d-md-block table-responsive">
+    <table class="table table-striped {% if not order_status_history %} hidden {% endif %}">
+        <thead>
+        <tr>
+            <th>{% trans %}State{% endtrans %}</th>
+            <th>{% trans %}User{% endtrans %}</th>
+            <th>{% trans %}Status{% endtrans %}</th>
+            <th>{% trans %}Description{% endtrans %}</th>
+            <th>{% trans %}Actions{% endtrans %}</th>
+        </tr>
+        </thead>
+        <tbody>
+        {% for status_history_item in order_status_history %}
+            <tr>
+                <td>{{ status_history_item.created_on|datetime }}</td>
+                <td>{{ status_history_item.creator }}</td>
+                <td>{{ status_history_item.next_order_status }}</td>
+                <td>{{ status_history_item.description if status_history_item.description else '-' }}</td>
+                <td>
+                    {% if not (status_history_item.order.is_complete() or status_history_item.order.is_canceled()) %}
+                    <a
+                        href="{{ url('shuup_admin:order.delete-order-status-history', pk=status_history_item.order_id) }}"
+                        data-order-status-history="{{ status_history_item.pk }}"
+                        name="delete-order-status-history-link"
+                        class="btn btn-danger btn-sm"
+                    >
+                        <i class="fa fa-close"></i> {{ _("Delete") }}
+                    </a>
+                    {% endif %}
+
+                </td>
+            </tr>
+        {% endfor %}
+        </tbody>
+    </table>
+</div>
+<div class="d-none d-sm-block d-md-none mobile-list-group {% if not order_status_history %} hidden {% endif %}">
+    <ul class="list-group">
+        {% for status_history_item in order_status_history %}
+            <li class="list-group-item">
+                <div class="row">
+                    <div class="col-sm-8">
+                        {{ status_history_item.created_on|datetime }}<br>
+                        {{ status_history_item.creator }}<br>
+                        {% trans %}Description{% endtrans %}: {{ status_history_item.description if status_history_item.description else '-' }}
+                    </div>
+                    <div class="col-sm-4 text-right">
+                        <strong>{{ status_history_item.next_order_status }}</strong>
+                    </div>
+                    <div class="col-xs-1 text-center">
+                        {% if not (status_history_item.order.is_complete() or status_history_item.order.is_canceled()) %}
```

Comment:
hummm I'm not sure we want to allow deleting order status changes :)

Reply:
I'll remove it.